netlify.toml

```diff
@@ -0,0 +1,25 @@
+# Sentry uses netlify to deploy pull-request previews of the frontend which
+# talk to the production API.
+#
+# This is an EXPERIMENTAL developer experience, there are still various pages
+# that are served through django and have no SPA equivalent views.
+
+[build]
+  publish = "src/sentry/static/sentry/dist"
+  command = "yarn webpack"
+
+[[redirects]]
+  from = "/_assets/*"
+  to = "/:splat"
+  status = 200
+
+[[redirects]]
+  from = "/api/*"
+  to = "https://sentry.io/api/:splat"
+  status = 200
+  force = true
```

Comment:
No need for `force` yet I think. We may even use the lack of force to put in mock API responses when/if we need.

You may also need to add this here ` headers = {Referer = "https://sentry.io/"}` as we do some CSRF checks w/ referrer.

Reply:
I'll add / remove these things when it's actually up and running so I can see where the problems are :)